client/js/components/statement/splitStatement/CardPaneCard.vue

```diff
@@ -53,10 +57,12 @@
       </span>
     </p>
 
-    <dp-button-icon
-      icon="fa-pencil"
+    <dp-button
```

Comment:
just a small styling nitpick: an additional space is needed here, because when these two buttons receive focus, the focus outline overlaps the adjacent button

Reply:
Thanks for testing it :heart: I fixed it in a way that the space is only added when focusing the buttons with the keyboard, since otherwise the space looks too big: a6769c970665fc639f25f141eeeee74a0980bbe3 and https://github.com/demos-europe/demosplan-ui/pull/1243

Reply:
unfortunately, I am not able to test it right now, but the idea is interesting, thx a lot